docs/developer-docs/latest/development/using-providers.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Using Providers - Strapi Developer Docs
+description: Install and use providers to extend the functionality of available plugins.
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/development/using-providers.html
+---
+
+# Using Providers
+
+Certain [plugins](../../../user-docs/latest/plugins/introduction-to-plugins.md) can be extended via the installation and configuration of additional [providers](../../../user-docs/latest/plugins/introduction-to-plugins.md#providers).
+
+Providers add an extension to the core capabilities of the plugin, for example to upload media files to Rackspace instead of the local server, or using Amazon SES for emails instead of Sendmail.
+
+::: note
+Only the [Upload](../plugins/upload.md) and [Email](../plugins/email.md) plugins are currently designed to work with providers. 
+:::
+
+For the relevant plugins, there are both official providers maintained by Strapi - discoverable via the [Marketplace](../../../user-docs/latest/plugins/installing-plugins-via-marketplace.md) - and many community maintained providers available via [npm](https://www.npmjs.com/).
+
+## Installing providers
+
+New providers can be installed using `npm` or `yarn` using the following format `@strapi/provider-<plugin>-<provider> --save`.
+
+For example:
+
+<code-group>
+
+<code-block title="npm">
```

Comment:
```suggestion
<code-block title="NPM">
```

Small exception here to what Mégane suggested, we use uppercase for code block titles :-)